src/pages/Admin/Core/Templates/registrar/Config/useConfigureRegistrar.ts

```diff
@@ -35,14 +35,14 @@ export default function useConfigureRegistrar() {
     //check for changes
     const diff = getPayloadDiff(initialConfigPayload, data);
     const diffEntries = Object.entries(diff) as [Key, Value][];
-    if (diffEntries.length <= 0) {
+    if (diffEntries.length === 0) {
       showModal(Popup, { message: "no changes detected" });
       return;
     }
+
     //convert presentational decimal to floating point string
     const finalPayload: RegistrarConfigPayload = {
       ...diff,
-      tax_rate: diff.tax_rate && `${+diff.tax_rate / 100}`,
```

Comment:
`split_default` changes were reverted as they still use the `"0"-"1"` range, but `tax_rate` does use `"0"-"100"` range?

Reply:
Tax rate is no longer in the Registrar Config.